src/osd/PG.cc

```diff
@@ -3776,7 +3776,7 @@ void PG::do_delete_work(ObjectStore::Transaction &t)
     max,
     &olist,
     &next,
-    0);
+    CEPH_OSD_OP_FLAG_FADVISE_WILLNEED | CEPH_OSD_OP_FLAG_FADVISE_SEQUENTIAL);
```

Comment:
put this in a separate patch?

also, should update the scrub caller with both of these flags...

Reply:
> put this in a separate patch?
done
> 
> also, should update the scrub caller with both of these flags...
not sure if this makes sense - currently prefetch is triggered for listings started from the beginning only.